src/components/API.js

```diff
@@ -0,0 +1,74 @@
+class Api {
```

Comment:
The `api.js` file should be in **UpperCase** like the other files, to make it consistent https://snipboard.io/qxtEbk.jpg



If the **file name** doesn't change on `github`, then you need to clear the `git cache` like this:

```bash

git rm -r --cached .
git add .
git commit -am 'git cache cleared'
git push

```